server/chat-commands/info.ts

```diff
@@ -715,21 +715,23 @@ export const commands: Chat.ChatCommands = {
 
 					if (move.isNonstandard === "Past" && dex.gen >= 8) details["&#10007; Past Gens Only"] = "";
 					if (move.secondary || move.secondaries) details["&#10003; Secondary effect"] = "";
-					if (move.flags['contact']) details["&#10003; Contact"] = "";
-					if (move.flags['sound']) details["&#10003; Sound"] = "";
-					if (move.flags['bullet']) details["&#10003; Bullet"] = "";
-					if (move.flags['pulse']) details["&#10003; Pulse"] = "";
+					if (move.flags['contact'] && gen >= 3) details["&#10003; Contact"] = "";
+					if (move.flags['sound'] && gen >= 3) details["&#10003; Sound"] = "";
+					if (move.flags['bullet'] && gen >= 6) details["&#10003; Bullet"] = "";
+					if (move.flags['pulse'] && gen >= 6) details["&#10003; Pulse"] = "";
 					if (!move.flags['protect'] && move.target !== 'self') details["&#10003; Bypasses Protect"] = "";
 					if (move.flags['bypasssub']) details["&#10003; Bypasses Substitutes"] = "";
 					if (move.flags['defrost']) details["&#10003; Thaws user"] = "";
-					if (move.flags['bite']) details["&#10003; Bite"] = "";
-					if (move.flags['punch']) details["&#10003; Punch"] = "";
-					if (move.flags['powder']) details["&#10003; Powder"] = "";
-					if (move.flags['reflectable']) details["&#10003; Bounceable"] = "";
+					if (move.flags['bite'] && dex.gen >= 6) details["&#10003; Bite"] = "";
```

Comment:
Why do some of these checks use `gen` and others `dex.gen`?

Reply:
...idk lol. probably just wasnt paying attention